Mentor - Meet & Greet Invite - Question Form is too narrow input det…

Thabo-hub opened the pull request:
When you log as an agent on Intern match and than click the plus button on the header to add another agent,
the form that is used to enter agent details uses the same card as the one on mentor match for Agree to mentor Group.

So on intern match the card is displayed nicely on width = 50%, but the one on mentor match it is displayed nicely when Width = full
So i create a ternary operator to decide the width for the card on mentor match and intern match.

Thabo-hub committed:
Mentor - Meet & Greet Invite - Question Form is too narrow input details

When you log as an agent on Intern match and than click the plus button on the header to add another agent,
the form that is used to enter agent details uses the same card as the one on mentor match for Agree to mentor Group.

So on intern match the card is displayed nicely on width = 50%, but the one on mentor match it is displayed nicely when Width = full
So i create a ternary operator to decide the width for the card on mentor match and intern match.

digesh-bajracharya reviewed (commented):
This will effect all the forms. Better if you can check a different property just for the detail-view-form.

Thabo-hub committed:
Merge branch '10.1.0' into Thabo-hub-patch-4

Thabo-hub committed:
Form Width

Fixing the width of the form. By not specifying the width it will adjust according to the screen width.

Thabo-hub commented:
I have removed the width property so the form will be responsive to the screen width and <center> element.

digesh-bajracharya committed:
fixed the form widths

digesh-bajracharya committed:
Merge branch '10.1.0' of https://github.com/genny-project/alyson into Thabo-hub-patch-4
